Library/src/Web/EmscriptenWebClient/EmscriptenWebClient.cpp

```diff
@@ -101,11 +102,26 @@ void OnFetchSuccessOrError(emscripten_fetch_t* Fetch)
 	emscripten_fetch_close(Fetch);
 }
 
+void OnFetchError(emscripten_fetch_t* Fetch)
+{
+	auto* UserData = reinterpret_cast<csp::web::HttpRequest*>(Fetch->userData);
+
+	if (UserData->Retry())
+	{
+		CSP_LOG_WARN_MSG("Retrying failed emscripten request\n");
+	}
+	else
+	{
+		OnFetchSuccessOrError(Fetch);
```

Comment:
This function name is now misleading. We should move the success code to a new `OnFetchSuccess` function, and move the error-related code to `OnFetchError`.

Reply:
Happy to leave this one as is if that's the case then.